src/main/java/uk/gov/hmcts/darts/common/entity/TranscriptionStatusEntity.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.hmcts.darts.common.entity;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.SequenceGenerator;
+import jakarta.persistence.Table;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Table(name = TranscriptionStatusEntity.TABLE_NAME)
+@Getter
+@Setter
+@EqualsAndHashCode(callSuper = false)
```

Comment:
I'm pretty sure that this can be removed as we haven't properly thought about implementing Equals / HashCode for our entities. I have a feeling it probably came about when I tried to introduce the `VersionedEntity` but the `version` column has since been dropped in the schema.

If you could remove it from all the Transcription* Entities please. We just want the Getter / Setter stuff for now.

FYI @cmagowan-hmcts / @hemantasharma1129